packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -449,6 +449,163 @@ curl -v -X GET \
 ]
 ```
 
+### List YubiKey OTP Tokens
+
+<ApiOperation method="get" url="/api/v1/org/factors/yubikey_token/tokens" />
+
+Enumerates all YubiKey OTP Tokens
+
+##### Request parameters
+
+| Parameter | Description                                                                                                      | Param Type | DataType | Required | Default |
+| --------- | ---------------------------------------------------------------------------------------------------------------- | ---------- | -------- | -------- | ------- |
+| after     | Specifies the pagination cursor for the next page of tokens                                                      | Query      | String   | FALSE    |         |
+| expand    | If YubiKey Token is assigned to a user and expand is set to `user`, embeds the [user](/docs/reference/api/users/#user-object) resource   | Query      | String   | FALSE    |         |
+| filter    | Filters tokens by `profile.email`, `profile.serial`, `activated`, `user.id`, `created`, `status`, or `lastVerified` expression | Query      | String   | FALSE    |         |
+| limit     | Specifies the number of results per page (maximum 200)                                                           | Query      | Number   | FALSE    | 20      |
+| sortBy    | Sorts the tokens by `profile.email`, `profile.serial`, `activated`, `user.id`, `created`, `status`, or `lastVerified` | Query      | String   | FALSE    |        |
+| sortOrder | Specifies the sort order, either `ASC` or `DESC`                                                                 | Query      | String   | FALSE    |         |
+| forDownload | Returns tokens in a CSV for download instead of in the response.  Defaults `limit` to 1000                     | Query      | Boolean  | FALSE    | false   |
```

Comment:
Period at the end:

Defaults `limit` to 1000.

Also - we alphabetize the parameters - so this item should move to just below the `filter` parameter.

Reply:
Thanks!  Do I need periods at the end of the other descriptions too?  Or just this one because it has multiple sentences?

Reply:
Just that one because it has multiple sentences. Thank you!